clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/HttpClientTransport.java

```diff
@@ -74,7 +75,7 @@ public void close() {
   public <T extends SpecificRecordBase, S extends SpecificRecordBase> T sendRequest(
       IngestionAction action,
       S param,
-      int timeoutMs) {
+      int requestTimeoutInSeconds) {
```

Comment:
Having the same name for a function param and class property is allowed by the language but can be confusing. Can we either remove the param (if we wish to use the class property) or name the param differently?

Reply:
Makes sense to me. Removed the argument and the method actually.